mon-pix/app/templates/fill-in-campaign-code.hbs

```diff
@@ -41,6 +41,12 @@
             </button>
           </div>
         </form>
+
+        {{#if this.isUserAuthenticated}}
+          <div class="fill-in-campaign-code__warning">
+            <span>{{this.warningMessage}} <span class="link" {{action this.disconnect}}>{{t 'common.actions.sign-out'}}</span></span>
```

Comment:
Il y a pas un espace en trop après `{{this.warningMessage}}` ?

Reply:
C'est pour éviter que le lien "Se déconnecter" soit collé au "?". Je préfères un "&nbsp;" ?

Reply:
ça marche